communication.md

```diff
@@ -191,3 +191,13 @@ Feedback is important to us. Knowing where we stand gives us clarity on
 what can be improved. We run period performance reviews based on peer
 feedback every three months. The primary tool are questionnaires, where
 we rate the work of our colleagues as well as our own.
+
+## Showing progress through heartbeats
+
+Showing progress is essential to keep the whole team informed and to spot projects that have gotten stuck. Read the chapter [Show Progress](https://basecamp.com/shapeup/3.4-chapter-13) from Shape up before continuing.
+
+We show the progress in the main GitHub issue of each project. The minimal progress update is an image of the current hill chart. We use [hillchart.app](https://hillchart.app/) to create and update them. We must copy an actual image into the issue, as this allows us to compare updates and see which scopes are moving and which are stuck.
```

Comment:
I assume that it is straightforward & self-explanatory how to prepare a chart in hillchart.app

Reply:
Yes. Did you get an invite? I added you to the Genistat team yesterday.

Reply:
yes, I registered already but I didn't try the app yet, that's why I asked 😅

Reply:
![](https://media.giphy.com/media/14smAwp2uHM3Di/giphy.gif) 

😄